plugins/org.eclipse.xtext/src/org/eclipse/xtext/formatting2/regionaccess/internal/NodeModelBasedRegionAccess.java

```diff
@@ -185,42 +155,61 @@ protected boolean include(INode node) {
 			return false;
 		}
 
-		protected boolean isComment(ILeafNode leaf) {
-			String text = leaf.getText();
-			for (int i = 0; i < text.length(); i++)
-				if (!Character.isWhitespace(text.charAt(i)))
+		protected boolean isDeadend(INode node) {
```

Comment:
I’ve deleted this method completely in 5d7d4ef77d984984e74fcdeb1dabf304d622bc50

On 27 Mar 2015, at 15:10, Sebastian Zarnekow <notifications@github.com> wrote:

> In plugins/org.eclipse.xtext/src/org/eclipse/xtext/formatting2/regionaccess/internal/NodeModelBasedRegionAccess.java:
> 
> > @@ -185,42 +155,61 @@ protected boolean include(INode node) {
> >  			return false;
> >  		}
> >  
> > -		protected boolean isComment(ILeafNode leaf) {
> > -			String text = leaf.getText();
> > -			for (int i = 0; i < text.length(); i++)
> > -				if (!Character.isWhitespace(text.charAt(i)))
> > +		protected boolean isDeadend(INode node) {
> should this be isDeadEnd ?
> 
> —
> Reply to this email directly or view it on GitHub.
>